en_US/LC_MESSAGES/django.po

```diff
@@ -4048,437 +4181,404 @@ msgstr "There is no topic data available for the current selections."
 #, fuzzy
 msgid "Modified {datetime} by {name}"
 msgstr "Modified by {name} on {datetime}"
 
-#: kitsune/questions/jinja2/questions/includes/answer.html:31 kitsune/questions/jinja2/questions/question_details.html:149
+#: kitsune/questions/jinja2/questions/includes/answer.html:75 kitsune/questions/jinja2/questions/question_details.html:180
 #, fuzzy
 msgid "Solution chosen by {user}"
 msgstr "Collections"
 
-#: kitsune/questions/jinja2/questions/question_details.html:149
+#: kitsune/questions/jinja2/questions/question_details.html:180
 #, fuzzy
 msgid "Chosen solution"
 msgstr "Collections"
 
-#: kitsune/questions/jinja2/questions/question_details.html:152
+#: kitsune/questions/jinja2/questions/question_details.html:187
 msgid "Read this answer in context"
 msgstr "Read this answer in context"
 
-#: kitsune/questions/jinja2/questions/mobile/question_details.html:38 kitsune/questions/jinja2/questions/question_details.html:164 kitsune/questions/jinja2/questions/question_details.html:507
+#: kitsune/questions/jinja2/questions/question_details.html:199 kitsune/questions/jinja2/questions/question_details.html:272
 #, fuzzy
 msgid "I have this problem, too"
 msgstr "1 person has this problem"
 
-#: kitsune/questions/jinja2/questions/question_details.html:176
+#: kitsune/questions/jinja2/questions/question_details.html:208
+#, fuzzy
+msgid "All Replies"
+msgstr "Replies"
+
+#: kitsune/questions/jinja2/questions/question_details.html:220
+#, python-format
+msgid ""
+"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
+"\">contributors forum</a> requesting that you be white-listed."
+msgstr ""
+"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
+"\">contributors forum</a> requesting that you be white-listed."
+
+#: kitsune/questions/jinja2/questions/email/new_answer.html:22 kitsune/questions/jinja2/questions/question_details.html:230
+#, fuzzy
+msgid "Post a Reply"
+msgstr "Post a reply"
+
+#: kitsune/questions/jinja2/questions/question_details.html:240
+msgid "I am providing the requested information"
+msgstr "I am providing the requested information"
+
+#: kitsune/questions/jinja2/questions/question_details.html:245
+#, fuzzy
+msgid "Needs more information"
+msgstr "Revision {num}"
+
+#: kitsune/questions/jinja2/questions/question_details.html:277
+msgid "You must <a href=\"{login_url}\">log in to your account</a> to reply to posts. Please <a href=\"{ask_url}\">start a new question</a>, if you do not have an account yet."
+msgstr "You must <a href=\"{login_url}\">log in to your account</a> to reply to posts. Please <a href=\"{ask_url}\">start a new question</a>, if you do not have an account yet."
```

Comment:
Grammar issue for the `Please ... ` part. remove the comma before `if`. Or if the sentence leads with `If, add a period before `please`.